src/ArborX_LinearBVH.hpp

```diff
@@ -58,6 +58,18 @@ class BoundingVolumeHierarchy
         return getBoundingVolume( getRoot() );
     }
 
+    Kokkos::View<bounding_volume_type, DeviceType> getBounds() const
+    {
+      Kokkos::View<bounding_volume_type, DeviceType> bounds("bounds");
+      using ExecutionSpace = typename DeviceType::execution_space;
+      auto bvh = *this;
+      Kokkos::parallel_for(
+          ARBORX_MARK_REGION( "copy bounds" ),
+          Kokkos::RangePolicy<ExecutionSpace>( 0, 1 ),
+          KOKKOS_LAMBDA( int ) { bounds() = bvh.bounds(); } );
```

Comment:
I am confused are you just doing a copy here? Why can't you use a `deep_copy`?

Reply:
I could but technically I only need to feel one entry in the vector.  Will change

Reply:
Oh it's not a vector you are 100% right sorry